content/guides/migrating-to-cypress/protractor.md

```diff
@@ -0,0 +1,872 @@
+
+## Parallelization
+
+One of the worst things that can happen to a developer is to be forced to wait for a 2 hour end-to-end test suite to before verifying something works or not. Instead, the [Cypress Dashboard Service](/guides/dashboard/introduction) allows your tests to run in parallel. If your longest test only test a minute to run, this means that you've just cut down your testing by over 12,000%!
+
+This feature is available in Protractor and requires you to configure your application with multiple options (i.e., `sharedTestFiles`, `maxInstances`, etc.). With Protractor your tests can be parallelized on a per-test basis. With Cypress, your tests can be [parallelized on a per spec file basis](https://docs.cypress.io/guides/guides/parallelization). This is an important distinction between Protractor and Cypress parallelization. One of the reasons why Cypress parallelizes tests per file because some users may write tests that build up state that subsequent tests, although we [do not recommend relying on the state of previous tests](/guides/references/best-practices#Having-tests-rely-on-the-state-of-previous-tests).
+
+An example is provided below:
+
+**Parallelization with Protractor:**
+
+```js
+// Example from https://developers.perfectomobile.com/display/PD/Protractor+parallel+execution
+// An example configuration file.
+exports.config = {
+  seleniumAddress:
+    'https://cloudName.perfectomobile.com/nexperience/perfectomobile/wd/hub',
+
+  // Capabilities to be passed to the webdriver instance.
+  capabilities: {
+    browserName: 'chrome',
+
+    // allows different specs to run in parallel.
+    // If this is set to be true, specs will be sharded by file
+    // (i.e. all files to be run by this set of capabilities will run in parallel).
+    // Default is false.
+    sharedTestFiles: true,
+
+    // Maximum number of browser instances that can run in parallel for this
+    // set of capabilities. This is only needed if shardTestFiles is true.
+    // Default is 1.
+    maxInstances: 2,
+
+    // Cloud capabilities
+    user: 'user@perfectomobile.com',
+    password: 'password',
+
+    // Device capabilities
+    platformName: 'Android',
+    manufacturer: 'Samsung',
+    model: 'Galaxy S5',
+  },
+
+  // Framework to use. Jasmine is recommended.
+  framework: 'jasmine',
+
+  // Spec patterns are relative to the current working directly when
+  // protractor is called.
+  specs: ['spec.js', 'spec2.js'],
+
+  // Options to be passed to Jasmine.
+  jasmineNodeOpts: {
+    showColors: true,
+    defaultTimeoutInterval: 30000,
+  },
+}
+```
+
+**Parallelization with Cypress:**
+
+However, with Cypress, all you need to do is pass the `--parallel` and `--record` flag to `cypress run`, and it will take care of the rest for you:
```

Comment:
```suggestion
With Cypress, all you need to do is pass the `--parallel` and `--record` flag to `cypress run`, and it will take care of the rest for you:
```